plugins/nrl-scoreboard/sports.py

```
@@ -698,23 +772,23 @@ def _load_fonts(self):
-            fonts["team"] = self._load_custom_font_from_element_config(team_config, default_size=8)
-            fonts["status"] = self._load_custom_font_from_element_config(status_config, default_size=6)
-            fonts["detail"] = self._load_custom_font_from_element_config(detail_config, default_size=6)
-            fonts["rank"] = self._load_custom_font_from_element_config(rank_config, default_size=10)
+            fonts["score"] = self._load_custom_font_from_element_config(score_config, default_size=10, element_key='score_text')
+            fonts["time"] = self._load_custom_font_from_element_config(period_config, default_size=8, element_key='period_text')
+            fonts["team"] = self._load_custom_font_from_element_config(team_config, default_size=8, element_key='team_name')
+            fonts["status"] = self._load_custom_font_from_element_config(status_config, default_size=6, element_key='status_text')
+            fonts["detail"] = self._load_custom_font_from_element_config(detail_config, default_size=6, element_key='detail_text')
+            fonts["rank"] = self._load_custom_font_from_element_config(rank_config, default_size=10, element_key='rank_text')
             self.logger.info("Successfully loaded fonts from config")
         except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)